Sources/Layout/MessagesCollectionViewLayoutAttributes.swift

```diff
@@ -40,15 +40,19 @@ open class MessagesCollectionViewLayoutAttributes: UICollectionViewLayoutAttribu
 
     public var cellTopLabelAlignment = LabelAlignment(textAlignment: .center, textInsets: .zero)
     public var cellTopLabelSize: CGSize = .zero
+    public var cellTopLabelPadding: UIEdgeInsets = .zero
```

Comment:
This is where we want the padding properties, so its confusing how there is a mixture of this and delegates. We should also have a convenience setter in `MessagesCollectionViewFlowLayout` to set this as we do with the other properties